### PR TITLE
Add Oakland recommendations 

### PR DESCRIPTION
**Pull request by brendenbattle**
This Pull Request closes issue #37.
### Adding Restaurants and activities
- Does anyone have a favorite time of motorcycles
- What part of Oakland is your favorite part of town?

@ninjaslice

### CODE REVIEW

**Comment by hectorsector** [comment]
@brendenbattle likes attention 🎉

File: githubteacher-Oakland.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Oakland :heart: :trophy: :trophy::trophy::trophy::trophy:
+
+#### Restaurants
+
+
+#### Things to do
```
Comment (hectorsector) [inline githubteacher-Oakland.md:6]: Are there things specific to Oakland that I should do while I'm there?

**Comment by beardofedu** [comment]
Giving you some comment ❤️

File: githubteacher-Oakland.md (hunk 1 above)
Comment (brendenbattle) [inline githubteacher-Oakland.md:6]: yeah brotha, thanks for the comment love! :)  Yes, you should come hang with me and grab a coffee at Kilovolt coffee in WEST OAKLAND.

**Pull request merged**